Jetsnack/app/src/main/java/com/example/jetsnack/ui/home/Cart.kt

```diff
@@ -16,20 +16,323 @@
+                )
+            }
+        )
+        IconButton(
+            onClick = { /* todo */ },
+            modifier = Modifier.constrainAs(remove) {
+                top.linkTo(parent.top)
+                end.linkTo(parent.end)
+            }.padding(top = 12.dp)
+        ) {
+            Icon(asset = Icons.Filled.Close, tint = JetsnackTheme.colors.brand)
+        }
+        Text(
+            text = snack.tagline,
+            style = MaterialTheme.typography.body1,
+            color = JetsnackTheme.colors.textHelp,
+            modifier = Modifier.constrainAs(tag) {
+                linkTo(
+                    start = image.end,
+                    startMargin = 16.dp,
+                    end = parent.end,
+                    endMargin = 16.dp,
+                    bias = 0f
+                )
+            }
+        )
+        Spacer(
+            Modifier
+                .preferredHeight(8.dp)
+                .constrainAs(priceSpacer) {
+                    linkTo(top = tag.bottom, bottom = price.top)
+                }
+        )
+        Text(
+            text = "$12.99",
+            style = MaterialTheme.typography.subtitle1,
+            color = JetsnackTheme.colors.textPrimary,
+            modifier = Modifier.constrainAs(price) {
+                linkTo(
+                    start = image.end,
+                    end = quantity.start,
+                    startMargin = 16.dp,
+                    endMargin = 16.dp,
+                    bias = 0f
+                )
+            }
+        )
+        QuantitySelector(
+            count = count,
+            updateCount = updateCount,
+            modifier = Modifier.constrainAs(quantity) {
+                baseline.linkTo(price.baseline)
+                end.linkTo(parent.end)
+            }
+        )
+        JetsnackDivider(
+            Modifier.constrainAs(divider) {
+                linkTo(start = parent.start, end = parent.end)
+                top.linkTo(parent.bottom)
+            }
+        )
+    }
+}
+
+@Composable
+fun SummaryItem(modifier: Modifier = Modifier) {
+    Column(modifier) {
+        Text(
+            text = stringResource(R.string.cart_summary_header),
+            style = MaterialTheme.typography.h6,
+            color = JetsnackTheme.colors.brand,
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis,
+            modifier = Modifier
+                .padding(horizontal = 24.dp)
+                .preferredHeightIn(min = 56.dp)
+                .wrapContentHeight()
+        )
+        Row(modifier = Modifier.padding(horizontal = 24.dp)) {
+            Text(
+                text = stringResource(R.string.cart_subtotal_label),
+                style = MaterialTheme.typography.body1,
+                modifier = Modifier.weight(1f)
+                    .wrapContentWidth(Alignment.Start)
+                    .alignWithSiblings(LastBaseline)
+            )
+            Text(
+                text = "\$27.47",
```

Comment:
Pass through the `item` rather than hardcoding values?

Reply:
That's planned for a next iteration. Currently the snacks have no associated price themselves, so calculating the total is not possible.

Reply:
Ack